cosmosdb/samples/Samples.md

```diff
@@ -0,0 +1,378 @@
+# Samples
+## Data-Plane operations
+## Documents
+Azure Cosmos DB allows the execution of CRUD operations on items separately. As this connector is using the Core API
+underneath, an item may refer to a document in the container. SQL API stores entities in JSON in a hierarchical
+key-value document. The max document size in Cosmos DB is 2 MB.
+
+### [Create a document](https://github.com/ballerina-platform/module-ballerinax-azure-cosmosdb/blob/main/samples/data-operations/documents/create_document.bal)
+This sample shows how to create a new document inside a container. This document contains information about a person's
+family and his/her gender. The container **my_container** has a partition key which has the path **/gender**. So, the
+document you create should contain a value for that path. Here, you have to provide *database ID*, *container ID* where
+the document will be created, A unique ID for the new document, and a JSON object which represents the document as
+parameters. As the partition key is made mandatory for the container, the value for that also should be passed as a
+parameter to the function.
+
+**Note:** <br/> Several Optional Parameters are supported in the creation of documents. These options can be specified in
+`DocumentCreateOptions` record type in the connector.
+- **indexingDirective** - This option is to specify whether the document is included in any predefined indexing policy
+  for the container. This is provided by giving **Include** or **Exclude**.
+- **isUpsertRequest** - You can convert the creation of a new document into an upsert request by using this parameter.
+  Must be a boolean value.
+  
+### [Replace document](data-operations/documents/replace_document.bal)
+This sample shows how to replace an existing document inside a container. Similar to document creation but, it replaces
```

Comment:
Suggestion: "This sample shows how to replace an existing document inside a container. This operation is similar to creating a document, but the new document replaces an existing one. The partition key of the new document needs to be the same as that of the document it is replacing. For more information about replacing partition key values, see [here](https://github.com/Azure/azure-sdk-for-js/issues/6324)."

Reply:
Replaced with this suggestion